CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ Features:
 - Add `k-u-margin` for margins.
 - Add modifier `k-buttonIcon--micro` on `k-ButtonIcon`.
 - Add `initialInstallment` option to `LoanSimulator`.
+- Display Utilities in the menu.
```

Comment:
J'aurais dit  `Show utilities in the styleguide`, pour bien faire comprendre de quel menu on parle.